plugin.video.iwm/.gitignore

```diff
@@ -0,0 +1,5 @@
+__pycache__/
```

Comment:
drop this file from the PR

Reply:
the whole .gitignore file not just this line

Reply:
Ah my bad :)